client/packages/coldchain/src/Equipment/ImportAsset/ImportReviewDataTable.tsx

```diff
@@ -24,11 +24,14 @@ export const ImportReviewDataTable: FC<ImportReviewDataTableProps> = ({
 }) => {
   const t = useTranslation();
   const isCentralServer = useIsCentralServerApi();
-  const [pagination, setPagination] = useState<Pagination>({
-    page: 0,
-    first: 20,
-    offset: 0,
-  });
+
+  const {
+    pagination: { page, first, offset },
+    updateUserPreferencePagination,
+  } = useUserPreferencePagination();
+
+  const pagination = { page, first, offset };
```

Comment:
```suggestion
  const {
    pagination,
    updateUserPreferencePagination,
  } = useUserPreferencePagination();
```

Looks like you're pulling these fields off just to rebuild an identical object - should be all good to pass pagination through directly!